front/src/common/BootstrapSNCF/NavBarSNCF.tsx

```diff
@@ -104,7 +104,7 @@ const LegacyNavBarSNCF = ({ appName, showLogoWithName }: Props) => {
         <li className="toolbar-item separator-gray-500">
           <DropdownSNCF
             titleContent={
-              <div className={cx(impersonatedUser ? 'impersonated-user' : 'd-none d-xl-block')}>
+              <div className={cx(impersonatedUser ? 'impersonated-user' : '')}>
```

Comment:
We can use this syntax here
```suggestion
              <div className={cx({ 'impersonated-user': impersonatedUser })}>
```

Reply:
Thanks @SharglutDev . I didn't know about this syntax. Much cleaner than what I had earlier :)